engine/src/main/scala/cromwell/engine/workflow/lifecycle/finalization/CopyWorkflowOutputsActor.scala

```diff
@@ -48,9 +48,22 @@ class CopyWorkflowOutputsActor(workflowId: WorkflowId, override val ioActor: Act
 
   private def copyWorkflowOutputs(workflowOutputsFilePath: String): Future[Seq[Unit]] = {
     val workflowOutputsPath = buildPath(workflowOutputsFilePath)
-
     val outputFilePaths = getOutputFilePaths(workflowOutputsPath)
 
+    // Check if there are duplicated destination paths and throw an exception if that is the case.
+    val destPaths = outputFilePaths.map {case (_ , dest) => dest}
+    val destPathOccurrences = destPaths.groupBy(identity).mapValues(_.length)
+    val duplicatedDestPaths = destPathOccurrences.filter{case (_, occurrences) => occurrences > 1}.keys.toSeq
+    if (duplicatedDestPaths.nonEmpty) {
+      val collidingCopyOptions = outputFilePaths.filter{
+        case (_ ,destination) => duplicatedDestPaths.contains(destination)
+          // Sort by destination path so collides are easy to spot in the exception message
+      }.sortBy { case (_, destination) => destination.pathAsString}
```

Comment:
I think you can simplify this logic somewhat. You can use this:
```scala
    val duplicatedDestPaths: Map[Path, Seq[Path]] = outputFilePaths.groupBy(_._2).collect {
      case (value, list) if list.size > 1 => value -> list.map(_._1)
    }
```

To immediately get a map from duplicated destinations to the paths which pointed at them